admin_guide/selfprovisioned_projects.adoc

```diff
@@ -10,31 +10,101 @@
 toc::[]
 
 == Overview
-In some cases, it is desirable to allow users to create projects for themselves without administrator intervention.  To make this simpler, a user accessible endpoint is present that will provision a project according to a template.  This endpoint is made accessible via: `osc new-project <project-name>`.
+You can allow developers to create their own projects. An accessible endpoint
+exists that will provision a project according to a
+link:../dev_guide/templates.html[template]. This endpoint is made accessible
+when a developer link:../dev_guide/projects.html[creates a new project].
 
-== Configuration
-=== New project template
-The API server creates new auto-provisioned projects based on a template that it finds based on the `projectRequestTemplate` property of the master-config.yaml.  The property is in the form "namespace/templatename", by default "openshift/project-request".  If the template does not exist, the API server will create a default template that creates a project with the requested name and assigns the requesting user to the "admin" role for that project.  If the property is empty, then no action is taken and self-provisioning will fail.
+== The New Project Template
```

Comment:
What about "Templates for New Projects"? Avoids starting off the title with "The".